peps/pep-0770.rst

```diff
@@ -179,98 +204,62 @@ Specification
 
 The changes necessary to implement this PEP include:
 
-* Additions to `Core Metadata <770-spec-core-metadata_>`_, as defined in the
-  `Core Metadata specification <coremetadataspec_>`__.
-* Additions to the author-provided
-  `project source metadata <770-spec-project-source-metadata_>`_, as defined in the
-  `pyproject.toml specification <pyprojecttoml_>`__.
-* `Additions <770-spec-project-formats_>`_ to the source distribution (sdist),
-  built distribution (wheel), and installed project specifications.
+* A new reserved registry of subdirectory names in the ``.dist-info`` directory.
+* A new reserved optional ``[additional-files]`` table with an optional
+  ``sboms`` key added to
+  `project source metadata <770-spec-project-source-metadata_>`_,
+  as defined in the `pyproject.toml specification <pyprojecttoml_>`__.
+* `Additions <770-spec-project-formats_>`_ to the built distribution (wheel),
+  and installed project specifications
 
 In addition to the above, an informational PEP will be created for tools
 consuming included SBOM documents and other Python package metadata to
 generate complete SBOM documents for Python packages.
 
-Terminology
------------
+.. _770-spec-dist-info-subdirs:
+
+Reserved ``.dist-info`` subdirectories registry
+-----------------------------------------------
+
+This PEP introduces a new registry of reserved subdirectory names allowed in
+the ``.dist-info`` directory for the :term:`distribution archive`
+and :term:`installed project` s project types. Future additions to this registry
+will be made through the PEP process. The initial values in this registry are:
+
+.. table::
```

Comment:
You don't need the ``.. table::`` directive in reST if just using a simple table.

You could consider the `list-table` approach though:

```rst
.. list-table::
   :align: left
   :header-rows: 1

   * - Directory name
     - PEP
   * - ``licenses``
     -  :pep:`639`
   * - ``license_files``
     -  N/A (See :ref:`770-backwards-compat`)
   * - ``LICENSES``
     -  N/A (See :ref:`770-backwards-compat`)
   * - ``sboms``
     -  :pep:`770`
```